src/frontend/js/components/Button/index.tsx

```diff
@@ -1,13 +1,20 @@
-import { FC } from 'react';
+import { DOMAttributes, FC } from 'react';
 
-interface ButtonProps {
-  color: 'primary' | 'secondary' | 'outline-primary' | 'outline-secondary';
+interface ButtonProps extends DOMAttributes<any> {
```

Comment:
More precise I think

```suggestion
interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
```

Reply:
This component may render a `<a/>` tag, I had the same idea though 😕